substrate-query-framework/index-builder/src/db/helper.ts

```diff
@@ -29,10 +29,12 @@ export function fillRequiredWarthogFields<T>(entity: DeepPartial<T>): DeepPartia
   return Object.assign(entity, requiredFields);
 }
 
-export async function createDBConnection(): Promise<Connection> {
+export async function createDBConnection(entities: any[] = []): Promise<Connection> {
```

Comment:
How do I pass warthog models here?

Reply:
Good point. I added `entities` field to `ProcessorOptions` which propagated down to here in `QueryNodeManager.process`. You can provide Warhog types to this field and they should be registered (but I haven't tested).

Reply:
👍